ci/pipelines/deploy-to-perf.yml

```diff
@@ -313,6 +350,49 @@ jobs:
         username: pay-concourse
         text: ":red_circle: Deployment of adminusers to perf failed - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>"
 
+  - name: adminusers-db-migration-perf
+    plan:
+      - get: pay-ci
+      - get: adminusers-db-ecr-registry-perf
+        params:
+          format: oci
+        trigger: true
+      - task: assume-role
+        file: pay-ci/ci/tasks/assume-role.yml
+        params:
+          AWS_ROLE_ARN: arn:aws:iam::((pay_aws_test_account_id)):role/concourse
+          AWS_ROLE_SESSION_NAME: terraform-test-assume-role
```

Comment:
Is this name arbitrary? I don't want to have to create a pay-infra PR to define another role session name. The role session names here are the same for an app deploy and db migration for the other deploy-to-[test/staging/prod] pipelines too so that seems to be the convention.

Reply:
Fixed